### DISCUSSION TITLE
help me batch split images using imagemagick

### DISCUSSION THREAD

**Discussion by MondayNitro** [ImageMagick/ImageMagick#7816] [Help]
### ImageMagick version

7.1.1-41

### Operating system, version and so on

windows 11

### Description

i have a folder with thousands of .jpg images that i want splitted from the centre (i.e left and right parts), on a single image the cmd command `magick input.jpg -crop 50%x100% +repage output_%d.jpg` works as intended but when i do the same in .bat script
```
for %%f in (*.jpg) do (
    magick "%%f" -crop 50%x100% +repage "%%~nf_%d.jpg"
)
```
doesnt work as intended. instead of splitting each 2000x2000 image into 1000x2000 images, each image is split into number of images of 50x2000 resolution.
i know i'm doing it wrong somewhere but i cant figure it out, please help me🙏

**Comment by snibgo** [comment]
You haven't doubled the percents in " -crop 50%x100%".
